elixir/apps/domain/priv/repo/migrations/20250214183914_create_internet_site.exs

```diff
@@ -0,0 +1,44 @@
+defmodule Domain.Repo.Migrations.CreateInternetSite do
+  use Ecto.Migration
+
+  def up do
+    execute("""
+    INSERT INTO gateway_groups (
+      id,
+      account_id,
+      name,
+      created_by,
+      managed_by,
+      inserted_at,
+      updated_at
+    )
+    SELECT
+      uuid_generate_v4(),
+      id,
+      'Internet',
+      'system',
+      'system',
+      NOW(),
+      NOW()
+    FROM accounts
+    WHERE deleted_at IS NULL
+    AND NOT EXISTS (
```

Comment:
In production we shouldn't have any of these, but in the `migrations and seeds test` this fails because the internet site is already created (in seeds)